### PR TITLE
[16.0][MIG] mrp_bom_version: Migration to 16.0

### PR DESCRIPTION
**Pull request by kaycross**

### CODE REVIEW

File: mrp_bom_version/models/mrp_bom.py [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+    operation_ids = fields.One2many(
+        readonly=True, states={"draft": [("readonly", False)]}
+    )
+    ready_to_produce = fields.Selection(states={"historical": [("readonly", True)]})
+    picking_type_id = fields.Many2one(states={"historical": [("readonly", True)]})
+    consumption = fields.Selection(states={"historical": [("readonly", True)]})
+    version = fields.Integer(
+        states={"historical": [("readonly", True)]}, copy=False, default=1
+    )
+    parent_bom = fields.Many2one(
```
Comment (matt454357) [inline mrp_bom_version/models/mrp_bom.py]: Suggestion: rename field `parent_bom` to `previous_bom`.  The name `parent_bom` seems confusing, given the hierarchical nature of a bill of materials.

Comment (kaycross) [reply mrp_bom_version/models/mrp_bom.py]: Thanks for the suggestion, change done

**Review by anajuaristi — APPROVED** [review]

File: mrp_bom_version/models/mrp_bom.py (hunk 1 above)
Comment (anajuaristi) [reply mrp_bom_version/models/mrp_bom.py]: @matt454357 @kaycross : is it ok for you to aprove? I would like to merge it but I would need more aprovals.
Thank you!!!

**Review by matt454357 — APPROVED** [review]
Looks good

**Pull request merged**